### PR TITLE
327 accordion UI block

### LINKED ISSUE #327: [Product Category] Accordion UI Block

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #327 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://327-accordion-ui-block--danaher-ls-aem--hlxsites.hlx.page/us/en/products/centrifuges

### CODE REVIEW

**Review by sharanagoudapatil — APPROVED** [review]
Changes looks good to me. Thanks.

File: blocks/accordion/accordion.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      { type: 'button', class: 'flex w-full items-start justify-between text-left text-gray-900' },
+      h3({ class: 'text-base font-semibold leading-7' }, question),
+      span({ class: 'ml-6 flex h-14 items-center pr-2' }),
+    ),
+  );
+  btn.querySelector('span').innerHTML = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" class="chevy ml-2 h-5 w-5 transition"><path fill-rule="evenodd" d="M12.53 16.28a.75.75 0 01-1.06 0l-7.5-7.5a.75.75 0 011.06-1.06L12 14.69l6.97-6.97a.75.75 0 111.06 1.06l-7.5 7.5z"></path></svg>';
+
+  const panel = dd(
+    { class: 'hidden panal mt-2 pr-12', id: panelId },
+    p({ class: 'text-base leading-7 text-gray-600 href-text v-html' }),
```
Comment (mhaack) [inline blocks/accordion/accordion.js:29]: `v-html` is a left over from old css

File: blocks/accordion/accordion.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  const divEl = div();
+  const btn = dt(
+    { id: buttonId, class: 'close button' },
+    button(
+      { type: 'button', class: 'flex w-full items-start justify-between text-left text-gray-900' },
+      h3({ class: 'text-base font-semibold leading-7' }, question),
+      span({ class: 'ml-6 flex h-14 items-center pr-2' }),
+    ),
+  );
+  btn.querySelector('span').innerHTML = '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" aria-hidden="true" class="chevy ml-2 h-5 w-5 transition"><path fill-rule="evenodd" d="M12.53 16.28a.75.75 0 01-1.06 0l-7.5-7.5a.75.75 0 011.06-1.06L12 14.69l6.97-6.97a.75.75 0 111.06 1.06l-7.5 7.5z"></path></svg>';
```
Comment (mhaack) [inline blocks/accordion/accordion.js:25]: `chevy` is a left over from old css, right?

**Review by mhaack — CHANGES_REQUESTED** [review]

File: blocks/accordion/accordion.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  activeButton.querySelector('span svg').classList.toggle('rotate-180', !isOpen);
+  panel.classList.toggle('hidden', isOpen);
+}
+
+function createAccordionBlock(question, answer, index) {
+  const buttonId = `button-${index}`;
+  const panelId = `panel-${index}`;
+  const divEl = div();
+  const btn = dt(
+    { id: buttonId, class: 'close button' },
```
Comment (duynguyen) [inline blocks/accordion/accordion.js:18]: If there're 2 accordion blocks on the same page, the button and panel IDs would be duplicate.
do we need ID here? if not let's remove it.

File: blocks/accordion/accordion.js (hunk 1 above)
Comment (rgravitvl) [reply blocks/accordion/accordion.js]: Removed

File: blocks/accordion/accordion.js (hunk 2 above)
Comment (rgravitvl) [reply blocks/accordion/accordion.js]: Removed

File: blocks/accordion/accordion.js (hunk 3 above)
Comment (rgravitvl) [reply blocks/accordion/accordion.js]: Removed Id

File: blocks/accordion/accordion.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import {
+  dl, dt, dd, div, h3, button, span, p,
+} from '../../scripts/dom-builder.js';
+
+function toggleAccordion(activeButton, panel) {
+  const isOpen = activeButton.classList.contains('open');
```
[the remark this answers is not in the document]
Comment (rgravitvl) [reply blocks/accordion/accordion.js:6]: where do we implemented aria-expanded before? I mean any reference?

[the remark this answers is not in the document]
Comment (mhaack) [reply blocks/accordion/accordion.js:6]: See https://github.com/hlxsites/moleculardevices/blob/main/blocks/accordion/accordion.js look for `openAttribute`

**Review by duynguyen — APPROVED** [review]

**Pull request merged**